### ISSUE TITLE
Error opening input file

### ISSUE THREAD

**Issue by Fu2chN** [crunchy-labs/crunchy-cli#389]
**Describe the bug**
Hi, I have "Error opening input files: Cannot allocate memory" which occurs in some title. The files doesn't merged and no trace.

**To Reproduce**
Steps / command to reproduce the behavior:
```
$ >"C:\Program Files\yt-dlp\crunchy-cli-v3.5.0.exe" archive --yes -a zh-CN -a ja-JP -a fr-FR -s fr-FR --merge audio --include-chapters --include-fonts --language-tagging ietf --skip-existing -o "E:\YTdlp\Crunchyroll\{series_name} - 
S{season_number}E{episode_number} - {title}.mkv" https://www.crunchyroll.com/fr/watch/GX9UQK7M1/guided-by-the-moon
```
**Expected behavior**
The file will be merged well.

**Screenshots**
![Sans titre](https://github.com/crunchy-labs/crunchy-cli/assets/13276983/79ad161d-b22c-4d29-a312-893e273b6443)

**Client (please complete the following information):**
 - OS: Windows 11 Pro
 - Version crunchy-cli 3.5.0

Thank for your help and powerfull tool.

**Comment by bytedream** [comment]
Thanks for reporting. This issue occurs because Crunchyroll reports that the intro ends at second 0 and starts at second 54, which is (understandably) behavior I didn't expect. But it got fixed in [v3.5.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.2) now

**Comment by Fu2chN** [comment]
> Thanks for reporting. This issue occurs because Crunchyroll reports that the intro ends at second 0 and starts at second 54, which is (understandably) behavior I didn't expect. But it got fixed in [v3.5.2](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.2) now

Ok tanh, I thought I had forgotten a synchro parameter to get through this problem.
